dev/io.openliberty.org.jboss.narayana.rts/src/io/openliberty/microprofile/lra/internal/CdiExtensionComponent.java

```diff
@@ -0,0 +1,46 @@
+package io.openliberty.microprofile.lra.internal;
+/*******************************************************************************
```

Comment:
better to put the copyright above the package statement

Reply:
moved